### ISSUE TITLE
[Feature Request]: more sidebar customization

### ISSUE THREAD

**Issue by JorgeR81** [Comfy-Org/ComfyUI_frontend#897]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

I have some ideas to improve the sidebar.
Some of these address similar problems, and may be redundant, so I decided to put them all on the same ticket.

- allow to bookmark folders ( e.g. Normal and Depth Estimators ), instead of all the nodes inside.

- save the last sidebar width. If possible, save a width value per use case ( node library and queue ). 

- smaller and more compact font option ( similar to the right click menu ).

- option to add "...", when necessary, instead of showing the node name in 2 lines.

- option to disable node preview, in the sidebar.

- make the circle icon ( in front of each node ) smaller.

---

![sidebar](https://github.com/user-attachments/assets/0e57fd1f-c25a-4ff5-b101-36a395fbbf64)

### Proposed workflow

.

**Comment by camoody1** [comment]
I would like to add a +1 for these requests:

- save the last sidebar width. If possible, save a width value per use case ( node library and queue ).
- smaller and more compact font option ( similar to the right click menu ).
- option to add "...", when necessary, instead of showing the node name in 2 lines.

The current version of the sidebar makes me resize it every time I reopen Comfy. These three changes would make the sidebar much more user-friendly.

**Comment by enternalsaga** [comment]
yes, +1 for smaller and more compact font option, the current sidebar is too big even in 'small' mode
